.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish a release
+
+on:
+  push:
+    tags:
+      - 'v**'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@master
+      
+      - name: Create a release
```

Comment:
Ditto.

Let me know what you think. Also. @fthouraud

Reply:
I choose use [softprops/action-gh-release@v1](https://github.com/softprops/action-gh-release) which is more stared